apps/meteor/app/emoji-custom/server/lib/uploadEmojiCustom.ts

```diff
@@ -41,9 +43,12 @@ export async function uploadEmojiCustomWithBuffer(
 		throw new Meteor.Error('not_authorized');
 	}
 
+	if (!Array.isArray(emojiData.aliases)) {
```

Comment:
In the previous revision of this file, the aliases were deleted without any checking, since it was always expected to be a string. Now, 
1. the data received from the insertOrUpdateEmoji is sent to this function. the data here will have aliases as array.
2. this function is called in emoji.create where aliases are query string.